.circleci/config.yml

```diff
@@ -481,21 +491,16 @@ workflows:
     when: << pipeline.parameters.generate_screenshots >>
     jobs:
       - raw-screenshots
-      - promo-screenshots
+      - promo-screenshots:
+          requires: [raw-screenshots]
```

Comment:
😱 Surprised we never had any issue before with this missing dependency… Nice catch!